lib/db/mysql.js

```diff
@@ -1138,14 +1155,20 @@ module.exports = function (log, error) {
           return resolve(connection)
         }
 
-        connection.query('SET NAMES utf8mb4 COLLATE utf8mb4_bin;', (err) => {
+        var mode = REQUIRED_SQL_MODES.join(',')
+        connection.query('SET SESSION sql_mode = \'' + mode + '\'', function(err) {
```

Comment:
can this use \` \` for the query here? might make it cleaner https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals